benchmarks/graphics_pipeline/main.cpp

```diff
@@ -111,20 +107,30 @@ class ProjApp
     using RenderList   = std::unordered_map<Material*, std::vector<Object*>>;
     using TextureCache = std::unordered_map<std::string, grfx::ImagePtr>;
 
-    std::vector<PerFrame>                           mPerFrame;
-    grfx::DescriptorPoolPtr                         mDescriptorPool;
-    grfx::DescriptorSetLayoutPtr                    mSetLayout;
+    std::vector<PerFrame>                                         mPerFrame;
+    grfx::DescriptorPoolPtr                                       mDescriptorPool;
+    grfx::DescriptorSetLayoutPtr                                  mSetLayout;
     std::array<grfx::ShaderModulePtr, kAvailableVsShaders.size()> mVsShaders;
     std::array<grfx::ShaderModulePtr, kAvailablePsShaders.size()> mPsShaders;
-    PerspCamera                                     mCamera;
-    float3                                          mLightPosition = float3(10, 100, 10);
-    BenchmarkSettings                               mBenchmarkSettings;
+    PerspCamera                                                   mCamera;
+    float3                                                        mLightPosition = float3(10, 100, 10);
 
     std::vector<Material>  mMaterials;
     std::vector<Primitive> mPrimitives;
     std::vector<Object>    mObjects;
     TextureCache           mTextureCache;
 
+private:
+    // Knobs
+    KnobStrDropdown*         pKnobVs = nullptr;
+    std::vector<std::string> knobChoicesVs;
```

Comment:
Do you need to store these choices here? From what I can see they are only used during knob creation